### PR TITLE
initial support for mojito (redmi note 10)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mojito.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "response": [
+    {
+        "maintainer": "Prince Cooper (pcooper)",
+        "oem": "Xiaomi",
```
Comment (gwolf2u) [inline mojito.json]: please set Redmi as oem

File: mojito.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "device": "Redmi Note 10",
+        "filename": "crDroidAndroid-11-20210603-mojito-7.6.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/mojito/7.x/crDroidAndroid-11-20210603-mojito-7.6.zip/download",
+        "timestamp": 1622705204,
+        "md5": "11d6501f76301b889bf6f4e057f6efb4",
+        "size": 1409252452,
+        "version": "7.6",
+        "buildtype": "Weekly",
+        "forum": "https://forum.xda-developers.com/t/rom-11-0-unofficial-crdroid-android-v7-6-mojito-sunny.4279593/",
+	"instructions": "https://telegra.ph/crDroid-Fastboot-ROM-Installation-for-Mojito-05-24",
```
Comment (gwolf2u) [inline mojito.json]: invalid tag, please use generated file only
don't alter json file

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Removed unneeded tag and set Xiaomi as OEM.

File: mojito.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "response": [
+    {
+        "maintainer": "Prince Cooper (pcooper)",
+        "oem": "OEM",
```
Comment (gwolf2u) [inline mojito.json]: Not oem, set Redmi here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**